src/client/api.cpp

```diff
@@ -132,6 +132,8 @@ py::dict getRuleAttributes(const prt::RuleFileInfo* ruleFileInfo) {
 		const std::wstring fullName(attr->getName());
 		if (fullName.find(L"Default$") != 0)
 			continue;
+		if (fullName.find(L"streetWidth") != std::wstring::npos && attr->getNumParameters() == 1)

```

Comment:
actually... just check

if (attr->getNumParameters() > 0)

the cga compiler only allows >0 parameters for an attribute with name "streetWidth" and 1 parameter which is float
and in general any attribute with parameters cannot be set because its actually a function where parameters must be provided
so this is a more general test and perfectly fine